### PR TITLE
clean up TxPb and Tx struct.

### PR DESCRIPTION
**Pull request by FishMeat**
#26

### CODE REVIEW

File: proto/blockchain.proto [hunk 1]
```diff
@@ -64,8 +38,11 @@ message VotePb {
 }
 
 message ActionPb {
+    uint32 version = 4;
```
Comment (zjshen14) [inline proto/blockchain.proto]: Remove L41 to L44

File: proto/blockchain.proto [hunk 2]
```diff
@@ -64,8 +38,11 @@ message VotePb {
...
+    uint64 nonce = 5;
+    bytes signature = 6;
+
     oneof action {
-        TxPb tx = 1; // To be deprecated
```
Comment (zjshen14) [inline proto/blockchain.proto]: Reorder the transfer and vote to 1 and 2. Otherwise the see/des will be broken.

**Comment by zjshen14** [comment]
Lgtm overall. Some minor comments

File: proto/blockchain.proto [hunk 3]
```diff
@@ -64,10 +38,13 @@ message VotePb {
 }
 
 message ActionPb {
+    uint64 nonce = 3;
```
Comment (zjshen14) [inline proto/blockchain.proto]: Can we remove L41-L44 from this PR? Let's keep this PR being a pure Tx and TxPb cleanup change.

**Comment by FishMeat** [comment]
removed.

**Comment by FishMeat** [comment]
the make failed, because removed L41-L44, it can't find version in actionPb.

**Comment by FishMeat** [comment]
the make failed, because removed L41-L44, it can't find version in actionPb.

**Comment by FishMeat** [comment]
modify overlay_test.go, remove version field

**Comment by zjshen14** [comment]
@FishMeat congrats on the first merged PR! Thanks for your contribution!

**Pull request merged**